src/main.ts

```diff
@@ -45,51 +45,69 @@ const args = require("yargs") // eslint-disable-line
     .describe("d", "date of work (default is todays date)")
     .default("d", `${today}`).argv;
 
+
 (async () => {
     try {
         const browser = await puppeteer.launch({
-            headless: !args.headmore,
-            userDataDir: DATA_PATH
+            // TODO: Waiting for the form submission is a bit flaky
+            headless: false,
```

Comment:
why remove the headless option?

Reply:
Hehe, yeah maybe too sunday commit without discussions.

For me the click on the submit button OR waiting for the form to complete doesn't seem to work. So if I run it headless I don't submit any form :P 

But yeah will update :)